installation_policies/pre_spack_install_policies.md

```diff
@@ -0,0 +1,124 @@
+This document lists the installation policies prior to [PR20](https://github.com/xsdk-project/xsdk-community-policies/pull/20), which changed policy M1 from requiring CMake/Autoconf installation to Spack based installation.
+
+----
+
+1. Implement the default behavior described below. Each package can decide whether XSDK mode is the default mode.<sup>4</sup>
+
+    1. `--dis/enable-xsdk-defaults`
+    2. `USE_XSDK_DEFAULTS=[YES,NO]`
+  
+<sup>4</sup> For packages like Trilinos that need to maintain backward compatibility over consecutive releases, 
+`USE_XSDK_DEFAULTS` may be `FALSE` by default.
```

Comment:
[I'm sure we discussed this before - and get confused with this].

I think its best to:
- remove mentioning this option [and requirement - the model pre-dates spack] 
- list what the defaults are
- If the default of the package default build is different [then what the xsdk default mode is] - it should provide a spack variant for it.

The current defaults I'm aware off are:
- shared
- debug
- double-precision
- 32bit-index

So they could be variants [as described further below]

Reply:
Correct, I consider an archive, and its no longer referenced in M1.

Reply:
However, I have suggested it could be updated and used as a guide for those using cmake/autoconf, but as you point out, it should then be changed. So may be the current document should be moved to an archive dir?